src/ansys/fluent/core/examples/downloads.py

```diff
@@ -135,8 +135,10 @@ def download_file(
 
 
 def path(filename: str):
-    file_path = str(Path(pyfluent.EXAMPLES_PATH) / filename)
-    if os.path.isfile(file_path):
-        return file_path
+    if os.path.isabs(filename):
+        return filename
+    file_path = Path(pyfluent.EXAMPLES_PATH) / filename
+    if file_path.is_file():
+        return str(file_path)
```

Comment:
@raph-luc Done. Thank you.

Reply:
Thanks @hpohekar, by the way we don't actually need this absolute path check, due to this:
![MicrosoftTeams-image](https://github.com/ansys/pyfluent/assets/132297401/8f09e7cd-c372-4ad4-bcf6-097c79ed6bda)

If `filename` is already an absolute path, then `Path(...) / filename` will just ignore whatever arguments are provided for `Path`. However, this is very unintuitive and error prone, so I think this is a good change for clarity and consistency.

Reply:
@raph-luc  Right, I have also verified this yesterday and compared output using both file name and file path. I just shared screenshot of output comparison but forgot to share following screenshot. 

![image](https://github.com/ansys/pyfluent/assets/106588300/1ebbefa3-b036-4f5d-8744-646ca84a5687)